sass/elements/_blocks.scss

```diff
@@ -15,3 +15,51 @@
 		}
 	}
 }
+
+hr {
+	border-bottom: 3px dotted $light-brown;
```

Comment:
I hadn't thought of just using the background for the dots. 🤦‍♂️ Got too caught up in getting the spacing between dots right and overcomplicated things.

Reply:
Actually my issue with a background image was still getting the spacing right and having it start on the far left and end on the far right at all sizes. I was trying to get 10px between each dot. I'll think about it a bit.

Reply:
@ryelle If you've got an idea for doing the dots as a background image I'm open to change this.

My thought process was I needed to get the far left and far right dots to always be at the edge and 10px between each dot without any scaling. So I needed that whitespace in the repeating SVG. So the SVG uses 2 dots one on the left, 10px, then one on the right. So I'm using the border-image-slice to get it to work. Though I don't have a lot of experience with that ruleset so there's some jank to it.

For now I dropped the `is-style-default` and applied those styles to the default `hr`, which I had meant to do on the original PR.

Let me know what you think.

Reply:
well, i've learned an awful lot about `border-image` tonight 😁The dots were still a bit too big, so I wanted to see how to tweak that.

It looks like since you're not actually wrapping the border around something, the corner "slices" are not applied. I've been playing around with it and tweaked the SVG, slice, and width values to get the size correct. I pushed that to this PR in 4926d9318f349659681bfb7df62c84332a7f8086 – the `dots` style will also need to be updated.

If you want to test it out border-image more, i recommend adding a different `fill` to each `circle` in the svg, then playing around with the different slice/width/etc values. For example, even though there's only a `border-bottom` on the element, it still tries to add the border image around the entire thing (hence updating it to `border-image-width: 0 0 4px`).